### DISCUSSION TITLE
mousewheel enable on vertical direction after Slider Section Stick on top

### DISCUSSION THREAD

**Discussion by shankyag** [nolimits4web/swiper#4396] [Q&A] [answered]
Hello Experts,

How Do I enable Mousewheel after .swiper-container sticky on top? Not before stick. 
ex: https://www.tldental.com.au/ (see the Second Section - _GOING TO...._). Here: Slides working after fixed on top.

Thank You.

**Comment by nolimits4web** [comment, marked as answer]
I think the logic should be the following:
- listen for `scroll` event on document (i.e. `document.addEventListener('scroll', handler)`
- in event handler check Swiper top offset (i.e. `document.querySelector('.swiper-container).getBoundingClientRect().top`)
- if top offset less than its sticky position (for example `<= 0` if it bound to top edge) do the `swiper.mousewheel.enable()`

**Reply by shankyag** [reply]
Yeah, I got it now.

But still, there is I am getting a bit issue. I am using position: sticky CSS on the parent. so when page scroll is finished and again re-scrolling after stick, that time only swiper mousewheel is working(not immediately).


```
document.addEventListener('scroll', function() {
	var sectionOffsetTop = document.getElementById("section-slider").getBoundingClientRect().top;
	if(sectionOffsetTop == 0 ){
		swiper.mousewheel.enable();
	} else {
		swiper.mousewheel.disable();
	}
});
```


Thank You @nolimits4web 

Yeah, there are no limits. :)
